core/src/main/scala/kafka/controller/TopicDeletionManager.scala

```diff
@@ -256,9 +257,6 @@ class TopicDeletionManager(config: KafkaConfig,
    * removed from their caches.
    */
   private def onTopicDeletion(topics: Set[String]): Unit = {
-    info(s"Topic deletion callback for ${topics.mkString(",")}")
```

Comment:
FYI - I have removed this one as well because the information is redundant with the log in `resumeDeletions`.